### ISSUE TITLE
Problem precompiling assets "Same file" error 

### ISSUE THREAD

**Issue by zkarpinski** [PerfectlyNormal/tinymce-rails-imageupload#72]
I ran into a problem identical to the following open issue in the tinymce-rails project: https://github.com/spohlenz/tinymce-rails/issues/183

In my reply to that issue I describe at least part of what causes the following crash:

```
Removing digest from public/assets/tinymce/plugins/uploadimage/langs/de-bef79730ed6c1394d01a609e466817ce.js
rake aborted!
public/assets/tinymce/plugins/uploadimage/langs/de-bef79730ed6c1394d01a609e466817ce.js and public/assets/tinymce/plugins/uploadimage/langs/de.js
~/.rvm/gems/ruby-2.1.5/gems/tinymce-rails-4.2.8/lib/tinymce/rails/asset_installer/copy.rb:42:in `block in move_asset'
```

The summary is that lib/tasks/tinymce-uploadimage-assets.rake probably needs to be updated to use the tinymce-rails configuration variable defined in application.rb (ie, :compile or :copy).

The fix is pretty simple but if you prefer I can issue a pull request for the following commit:
https://github.com/zkarpinski/tinymce-rails-imageupload/commit/06db01685a708c84625d070f38ad1e74628d07a9

**Comment by sobrinho** [comment]
Hello @zkarpinski, can you open a pull request?

**Comment by botimer** [comment]
This was resolved by pull request #77.

**Comment by PerfectlyNormal** [comment]
Thanks!
Try with v4.0.17.beta.3, which should fix it
